src/dicom/dicomParser.js

```diff
@@ -1287,36 +1289,75 @@ dwv.dicom.DicomParser.prototype.parse = function (buffer)
     offset = 128;
     var magicword = metaReader.readString( offset, 4 );
     offset += 4 * Uint8Array.BYTES_PER_ELEMENT;
-    if(magicword !== "DICM")
+    if(magicword !== "DICM") // Check for Dicom file without metadata: attempt to detect transfer syntax
     {
-        throw new Error("Not a valid DICOM file (no magic DICM word found)");
-    }
+        offset = 0;
+        dataElement = this.readDataElement(dataReader, offset, implicit);  // read first element
+        if ((dataElement.tag.group != "0x0008") && (dataElement.tag.group != "0x0800")) {  // check character set
```

Comment:
Not sure about the second group check on this line, do you mean element? Something like:
```suggestion
        if ((dataElement.tag.group != "0x0008") && (dataElement.tag.element != "0x0018")) {  // check character set
```

Reply:
Because there is no metadata, the endianness is unknown, and the first group can be either 0x0800 (for a big endian file) or 0x0008 (for a little endian file).

Reply:
Ok, now I get it, thanks!